tests/cypress/integration/skeletons_pipeline.js

```diff
@@ -184,7 +212,7 @@ context('Manipulations with skeletons', () => {
             cy.removeAnnotations();
         });
 
-        it('Splitting two skeletons and merge them back', () => {
+        it.skip('Splitting two skeletons and merge them back', () => {
```

Comment:
Is this test broken? For me it fails locally.
![image](https://user-images.githubusercontent.com/50956430/188608114-89a0009d-6fdd-4f7a-989a-d3eea90cc7db.png)

Reply:
This error has been fixed here #4892, I do not have any ideas about why you see it now.
I will run /check command
Locally it passes for me

![image](https://user-images.githubusercontent.com/40690378/188613669-67a2636a-91f7-494d-abee-e98e734ee3c7.png)

Reply:
Ok, i see CI is green, it seems something on my side